docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -61,6 +61,8 @@ Property Name                                      Required   Description
                                                               connection properties.  These files must exist on the machines running Trino.
 ================================================== ========== ===================================================================================
 
+.. include:: jdbc-common-configurations.fragment
```

Comment:
This will be awkward in the context of this doc; The section above has a configuration properties table, introduced with "The following configuration properties are available. This fragment adds a second configuration properties table, introduced with "The following table describes general configuration properties for the connector."

Propose changing the fragment introduction to something like "The following table describes configuration properties that are available for this and other JDBC-based connectors" which at least distinguishes it a little bit from the above table.

Or, you can update the introduction on line 50 to say something like "The following Phoenix-specific configuration properties are available".

Reply:
+1

Reply:
The latter option makes more sense to me since the fragment applies to all JDBC connectors and `for this and other JDBC-based connectors` sounds weird when I'm looking at docs for a single JDBC connector.

Reply:
Changed. PTAL. Also removed the double `Configuration` and `Configuration properties` redundancy in the Phoenix docs. This is how it looked earlier 
![image](https://user-images.githubusercontent.com/10473142/127891390-e9a1cd41-c046-4df5-b635-25f111bea309.png)